terraform/cross-device/spanner.ddl.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE Task(PopulationName STRING(64) NOT NULL, TaskId INT64 NOT NULL, TotalIteration INT64, MinAggregationSize INT64, MaxAggregationSize INT64, Status INT64, CreatedTime TIMESTAMP, StartTime TIMESTAMP, StopTime TIMESTAMP, StartTaskNoEarlierThan TIMESTAMP, DoNotCreateIterationAfter TIMESTAMP, MaxParallel INT64, CorrelationId STRING(MAX), MinClientVersion STRING(32), MaxClientVersion STRING(32)) PRIMARY KEY(PopulationName,TaskId);
```

Comment:
Nit: can you move this file in a `files` directory under the `cross-device` module? So we don't mix it with `tf` files.